src/core/room/room.server.ts

```diff
@@ -105,7 +105,12 @@ export class ServerRoom extends Room<WorkPlace.Server> {
   }
 
   private initAIPlayers() {
-    if (this.gameMode === GameMode.Pve && this.Players.length === 0) {
+    if (this.gameMode === GameMode.Pve && [4, 5].includes(this.gameInfo.numberOfPlayers)) {
```

Comment:
anyway to avoid using if else here? we can directly use `numberOfPlayers` to add smart players for all pve mode

Reply:
We have to judge the number of players, I don't know how to avoid this judgement